code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -392,6 +392,115 @@
 
 	return 1
 
+/datum/reagent/stableslimetoxin
+	name = "Human Mutation Toxin"
+	id = "humanmutationtoxin"
+	description = "A humanizing toxin produced by slimes."
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	metabolization_rate = INFINITY //So it instantly removes all of itself
+	var/datum/species/race = /datum/species/human
+	var/mutationtext = "<span class='danger'>The pain subsides. You feel... human.</span>"
+
+/datum/reagent/stableslimetoxin/on_mob_life(mob/living/carbon/human/H)
+	..()
+	H << "<span class='warning'><b>You crumple in agony as your flesh wildly morphs into new forms!</b></span>"
+	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
+	H.Weaken(3, 0)
+	spawn(30)
+		if(!H || qdeleted(H))
+			return
+
+		var/current_species = H.dna.species.type
+		var/datum/species/mutation = race
+		if(mutation && mutation != current_species)
+			H << mutationtext
+			H.set_species(mutation)
+		else
+			H << "<span class='danger'>The pain vanishes suddenly. You feel no different.</span>"
+
+	return 1
```

Comment:
i could also make the normal slime toxin a child of stable, it seems its transformation is currently hardcoded into the human species_types